db/version_set.cc

```diff
@@ -38,14 +68,16 @@ static int64_t ExpandedCompactionByteSizeLimit(const Options* options) {
   return 25 * TargetFileSize(options);
 }
 
-static double MaxBytesForLevel(const Options* options, int level) {
+static double MaxBytesForLevel(const Options* options, int level, int max_level_in_use) {
   // Note: the result for level zero is not really used since we set
   // the level-0 compaction threshold based on number of files.
 
-  // Result for both level-0 and level-1
+  // Result for level-0
```

Comment:
Maybe we should shrink this factor?
Preivously, it is assuming L1 is 10 MB which implies L0 is about 1 MB.
We should probably reduce result to `double result = 1048576.0;`